astro_2.0/components/TokenWidget/TokenWidget.tsx

```diff
@@ -8,28 +8,30 @@ import styles from './TokenWidget.module.scss';
 interface TokenWidgetProps {
   amount: string;
   icon: string;
+  noIcon?: boolean;
   symbol: string;
   decimals: number;
 }
 
 export const TokenWidget: React.FC<TokenWidgetProps> = ({
   amount,
+  noIcon = false,
   icon,
   symbol,
   decimals,
 }) => {
   const isValid = useIsValidImage(icon);
 
   function renderIcon() {
```

Comment:
Maybe, we can make it in a following way
```
function renderIcon() {
  if (!noIcon) {
    if (symbol === 'NEAR') {
      return (
        <div className={styles.icon}>
          <Icon name="iconNear" width={24} />
        </div>
      );
    }

    if (isValid) {
      return (
        <div
          style={{
            backgroundImage: `url(${icon})`,
          }}
          className={styles.icon}
        />
      );
    }
  }
   
    return <div className={styles.icon} />;
}
```

Reply:
fixed